app/views/registration/_form.erb

```diff
@@ -0,0 +1,74 @@
+    </div>
+  </div>
+  <div class="form-group ">
+    <label class="col-md-2 control-label"><%= _('Location') %></label>
+    <div class="col-md-4">
+      <%= select_tag 'location', options_from_collection_for_select(@locations, :id, :name, params[:location]), class: 'form-control' %>
+    </div>
+  </div>
+  <div class="form-group ">
+    <label class="col-md-2 control-label"><%= _('Host Group') %></label>
+    <div class="col-md-4">
+      <%= select_tag 'host_group', options_from_collection_for_select(@host_groups, :id, :name, params[:host_group]), class: 'form-control', include_blank: '' %>
+    </div>
+  </div>
+  <div class="form-group ">
+    <label class="col-md-2 control-label"><%= _('Operating System') %></label>
+    <div class="col-md-4">
+      <%= select_tag 'operating_system', options_from_collection_for_select(@operating_systems, :id, :title, params[:operating_system]), class: 'form-control', include_blank: '' %>
+    </div>
+  </div>
+  <div class="form-group ">
+    <label class="col-md-2 control-label"><%= _('Proxy') %></label>
+    <div class="col-md-4">
+      <%= select_tag 'smart_proxy', options_from_collection_for_select(@smart_proxies, :id, :name, params[:smart_proxy]), class: 'form-control', include_blank: '' %>
+    </div>
+  </div>
+  <div class="form-group ">
+    <label class="col-md-2 control-label"><%= _('JWT Expiration') %>&nbsp;<%= popover('', _('Expiration of JWT authorization token in hours'), options = {}) %></label>
+    <div class="col-md-4">
+      <%= number_field_tag 'jwt_expiration', 4, class: 'form-control', min: 1 %>
+    </div>
+  </div>
+  <% @plugin_params.each do |pr| %>
+    <div class="form-group ">
+      <label class="col-md-2 control-label"><%= pr.to_s.titleize %></label>
+      <div class="col-md-4">
+        <%= text_field_tag pr, params[pr], class: 'form-control' %>
+      </div>
+    </div>
+  <% end%>
+  <div class="form-group ">
+    <%= submit_tag _("Submit"), :class => "btn btn-primary" %>
+    <%= link_to _("Cancel"), hosts_path, :class => "btn btn-default" %>
+  </div>
+
+  <% if action_name == 'create' %>
+  <script type="text/javascript">
+    function register_copy(text) {
+      var aux = document.createElement("textarea");
+      $(aux).html(text);
+      document.body.appendChild(aux);
+      aux.select();
+      document.execCommand("copy");    <%= submit_tag _("Submit"), :class => "btn btn-primary" %>
+    <%= link_to _("Cancel"), hosts_path, :class => "btn btn-default" %>
+      document.body.removeChild(aux);
+    }
+  </script>
+    <div class="form-group ">
+      <label class="col-md-2 control-label"><%= _('Command') %></label>
+      <div class="col-md-8">
+        <pre class="ellipsis white-space-normal" id="registration_command">curl -X GET "<%= @endpoint %>?<%= @registration_args.to_query %>" <%= @headers %> | bash</pre>
```

Comment:
Nit: should the controller provide a complete URL? IMHO it's not something you'd do in a view